### PR TITLE
About us page modified

### PR DESCRIPTION
**Pull request by rakesh-201**
The Team section of the about us page earlier had collages. This is now replaced with individual standalone image cards

### CODE REVIEW

**Review by akshatshah21 — CHANGES_REQUESTED** [review]

File: new_client/src/components/about-us/details.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
+export default [
```
[the remark this answers is not in the document]
Comment (rakesh-201) [reply new_client/src/components/about-us/details.js:1]: Ohh yes; I'll do that my tomorrow

**Comment by rakesh-201** [comment]
The requested changes are made!

File: new_client/src/components/about-us/AboutUs.jsx [hunk 2]
```diff
@@ -14,10 +14,27 @@ import GenericDetails from "./GenericDetails";
...
 export default function AboutUs() {
+  const details = Details;
+  const arr = [];
+
+  for (let i = 0; i < 30; i += 2) {
+    arr.push(
+      <div class="com-mem">
+        <Image class="member" name={details[i].name} img={details[i].img} />
+        <Image
+          class="member"
```
Comment (ShubhankarKG) [inline new_client/src/components/about-us/AboutUs.jsx:30]: For this and all other instances in this component, please update `class` to `className`.

File: new_client/src/components/about-us/AboutUs.jsx [hunk 3]
```diff
@@ -195,26 +212,28 @@ export default function AboutUs() {
             <br />
             {/* <Grid container> */}
             {/* <Grid item xs={0} md={4} lg={4} /> */}
+
             <div style={{ textAlign: "center" }}>
+              <Typography variant="h4">General Secretaries</Typography>
+              <div class="gene-sec-cont">
```
Comment (ShubhankarKG) [inline new_client/src/components/about-us/AboutUs.jsx:218]: Please update `class` to `className`

File: new_client/src/components/about-us/Image.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import GSes from "../assets/coc-gses.webp";
+
+const Image = (props) => {
+  return (
+    <div class={`image-cont ${props.class}`}>
+      <div class={`image-card ${props.class}`}>
+        <img src={props.img} class="image" height="100%" width="100%" />
+      </div>
+      <div class="image-name">{props.name}</div>
```
Comment (ShubhankarKG) [inline new_client/src/components/about-us/Image.js:10]: Please update all instances of `class` to `className`

File: new_client/src/components/about-us/AboutUs.jsx [hunk 5]
```diff
@@ -14,10 +14,27 @@ import GenericDetails from "./GenericDetails";
...
-
 import "./AboutUs.css";
+import Image from "./Image";
+import Details from "./details";
 
 export default function AboutUs() {
+  const details = Details;
+  const arr = [];
+
+  for (let i = 0; i < 30; i += 2) {
```
Comment (ShubhankarKG) [inline new_client/src/components/about-us/AboutUs.jsx:25]: This increment will not work if you have an odd number of members.

1. Please don't hard code the values. If we have the data, we know its length. Please use Array.length instead.
2. Please update this logic to accommodate the last person who is left out.

File: new_client/src/components/about-us/AboutUs.jsx [hunk 6]
```diff
@@ -14,10 +14,27 @@ import GenericDetails from "./GenericDetails";
...
+    arr.push(
+      <div class="com-mem">
```
Comment (ShubhankarKG) [inline new_client/src/components/about-us/AboutUs.jsx:27]: Please update this to `className`

File: new_client/src/components/about-us/details.js [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+    img:
+      "https://res.cloudinary.com/dxleddac7/image/upload/v1622393499/DevanshiShah_kwo6aq.jpg",
+  },
+  {
+    name: "Atharva Thorve",
+    img:
+      "https://res.cloudinary.com/dxleddac7/image/upload/v1622393498/Atharva_Thorve_uzssqr.jpg",
+  },
+  {
+    name: "archeel Parikh",
```
Comment (ShubhankarKG) [inline new_client/src/components/about-us/details.js:153]: Archeel Parekh.

This is the item that gets left out in the loop.

File: new_client/src/components/about-us/details.js [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+    img:
+      "https://res.cloudinary.com/dxleddac7/image/upload/v1622393523/Siddhesh_Bandgar_v3anih.jpg",
+  },
+  {
+    name: "Aditya Patkar",
+    img:
+      "https://res.cloudinary.com/dxleddac7/image/upload/v1622393522/Aditya_Patkar_dd8hyh.png",
+  },
+  {
+    name: "Sarakshi ",
```
Comment (ShubhankarKG) [inline new_client/src/components/about-us/details.js:48]: Sarakshi Phate

**Review by ShubhankarKG — CHANGES_REQUESTED** [review]
Apart from the inline changes, please note the following changes.

1. The images have lost aspect ratio. Please use cloudinary's query paras features to ensure that the images don't appear to be stretched.
2. The member's list is in no particular order. Please sort all members on their names alphabetically.

File: new_client/src/components/about-us/AboutUs.jsx (hunk 2 above)
Comment (rakesh-201) [reply new_client/src/components/about-us/AboutUs.jsx]: The Image Component is a custom component, so it's ok if I use class, right?

**Comment by rakesh-201** [comment]
All the requested changes are implemented.

**Comment by rakesh-201** [comment]
All the requested changes are implemented.

**Review by ShubhankarKG — APPROVED** [review]

**Pull request merged**